_news/19.md

```diff
@@ -11,7 +11,7 @@ visible: "true"
 article_type_id: "3"
 ---
 
- We are aware that parts of erlang.org need improvement. For example <https://www.erlang.org/article/tag/examples> and <https://www.erlang.org/course/course.html> are outdated. We would like to see a number of small code examples for beginners. The purpose of these examples is to provide an attractive and useful introduction for people who are interested in adopting the Erlang programming language. 
+ We are aware that parts of erlang.org need improvement. For example </article/tag/examples> and <https://erlang.org/course/course.html> are outdated. We would like to see a number of small code examples for beginners. The purpose of these examples is to provide an attractive and useful introduction for people who are interested in adopting the Erlang programming language. 
```

Comment:
`/article/tag/examples` was removed in the new erlang.org version, so keep this as a dead link.

Reply:
Fixed.